libvast/test/system/exporter.cpp

```diff
@@ -183,7 +185,8 @@ TEST(continuous query with exporter only) {
   send(exporter, system::extract_atom::value);
   sched.run();
   MESSAGE("send conn.log directly to exporter");
-  vast::detail::spawn_container_source(sys, bro_conn_log, exporter);
+  vast::detail::spawn_container_source(sys, const_bro_conn_log_slices,
+                                       exporter);
```

Comment:
Why do you copy the conn log slices above and not here? Because they are const?

Reply:
@mavam I've added this comment to where we make the slice copies:

```
  // We need to copy bro_conn_log_slices here, because the importer will assign
  // IDs to the slices it received and we mustn't mess our static test data.
```

I hope that clears things up.

Reply:
Perfect, thanks!